coderd/audit.go

```diff
@@ -197,34 +199,112 @@ func convertAuditLog(dblog database.GetAuditLogsOffsetRow) codersdk.AuditLog {
 		Diff:             diff,
 		StatusCode:       dblog.StatusCode,
 		AdditionalFields: dblog.AdditionalFields,
-		Description:      auditLogDescription(dblog),
 		User:             user,
+		Description:      auditLogDescription(dblog),
+		ResourceLink:     api.auditLogResourceLink(ctx, dblog),
```

Comment:
Because this calls `api.auditLogIsResourceDeleted` as well, you are making the same database calls twice here. One way to speed this up and save the work would be to call `api.auditLogIsResourceDeleted` before this struct and pass it into both `IsDeleted` and also as an arg into `api.auditLogResourceLink`.

Reply:
Woops, you're totally right! Pushing a fix now.